packages/spark-core/src/index.js

```diff
@@ -40,7 +40,7 @@ export {
 export {
   SparkPluginStorage,
   Store,
-  MemoryStoreAdapter,
+  // MemoryStoreAdapter,
```

Comment:
There are still two sets of exports from `./lib/storage`; they should be combined.